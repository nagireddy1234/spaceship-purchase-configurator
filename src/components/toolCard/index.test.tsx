import React from 'react';
import renderer from 'react-test-renderer';
import ToolCard from './index';

it('Checking ColorSelector component rendring correctly or not.', () => {
    const home = renderer.create(<ToolCard title="100 MW" price={0} selected={false} onClick={() => {}} />).toJSON();
    expect(home).toMatchSnapshot();
});
