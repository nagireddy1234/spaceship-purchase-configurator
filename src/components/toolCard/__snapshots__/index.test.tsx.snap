// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking ColorSelector component rendring correctly or not. 1`] = `
<div
  className="col-4 "
  onClick={[Function]}
>
  <p
    className="title"
  >
    100 MW
  </p>
  <p
    className="titlePrice"
  >
    +
    0
    â‚¬
  </p>
</div>
`;
